eng/pipelines/linux.yml

```diff
@@ -107,21 +117,21 @@ jobs:
                 _buildScriptPrefix: ''
                 _buildExtraArguments: /p:RuntimeOS=linux-musl
 
-            musl_arm64_Release:
-              _BuildConfig: Release
-              _architecture: arm64
-              _framework: netcoreapp
-              _dockerContainer: alpine_37_arm64_container
-              _buildScriptPrefix: 'ROOTFS_DIR=/crossrootfs/arm64 '
-              _buildExtraArguments: -warnAsError false /p:BuildNativeClang=--clang5.0 /p:RuntimeOS=linux-musl
+              musl_arm64_Release:
```

Comment:
Why did you make `musl_arm64_Release` to only run on CI? You should leave as it was, and just move `arm_Release` to before musl, or move musl leg to before the if statement so that it is clear that it should run on both, official and CI.

Reply:
I'll take another look. The indentation did not look right but maybe I misses something...

Reply:
updated.